contrib/bradfitz/gomemcache/memcache/option.go

```diff
@@ -57,3 +58,10 @@ func WithAnalyticsRate(rate float64) ClientOption {
 		}
 	}
 }
+
+// WithValueTags enables tracing of values used in operations
+func WithValueTags(on bool) ClientOption {
```

Comment:
Can we please make some changes here?
```suggestion
// WithValueTags specifies whether values assigned to keys in memcache operations should be
// added to spans as tags.
func WithValueTags() ClientOption {
```
We don't need it to be a `bool`. If it's specified it will be on, and if it's not specified it will be off.

Reply:
Got it. I fixed it. (+ I moved the `should be` text to the second line)